source/_docs/automation/trigger.markdown

```diff
@@ -892,6 +892,32 @@ automation:
 See the [Calendar](/integrations/calendar/) integration for more details on event triggers and the
 additional event data available for use by an automation.
 
+## Sentence trigger
+
+A sentence trigger fires when [Assist](https://www.home-assistant.io/voice_control/) matches a sentence from a voice assistant, such as [ESPHome](https://www.home-assistant.io/voice_control/thirteen-usd-voice-remote/) or an [analog phone](https://www.home-assistant.io/voice_control/worlds-most-private-voice-assistant/).
```

Comment:
I wouldn’t mention implementations of Assist. Leave that up to the page that Assist links to. I would mention that it requires the Home Assistant conversation agent.

Reply:
Linking to the conversation integration now